src/main/java/uk/ac/ebi/eva/pipeline/configuration/validation/DbCollectionsVariantsNameValidator.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.validation;
+
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersInvalidException;
+import org.springframework.batch.core.JobParametersValidator;
+
+import uk.ac.ebi.eva.pipeline.configuration.JobParametersNames;
+
+public class DbCollectionsVariantsNameValidator implements JobParametersValidator{
```

Comment:
`ConfigRestartabilityAllowValidator` included a Javadoc string in the class header, but the other validators document the `validate` method.

In order to keep the format of the documentation consistent, I think we should prefer to use the class documentation; the `validate` method [default documentation](http://docs.spring.io/spring-batch/apidocs/org/springframework/batch/core/JobParametersValidator.html) is still useful.